.babelrc

```diff
@@ -47,6 +48,11 @@
           }
         ]
       ]
+    },
+    "test": {
+      "plugins": [
+        "dynamic-import-node"
```

Comment:
why import `dynamic-import-node`?,  transpile for nodejs ?

Reply:
Dynamic import for jest